tests/poiseuille-2d-horizontal-pressure-bc.prm

```diff
@@ -0,0 +1,107 @@
+# Like the poiseuille-2d.prm test, but instead of prescribing the
+# velocity at the left and right ends, prescribe the horizontal
+# component of the traction. At the same time, set the vertical
+# component of the velocity to zero. This should yield the same answer.
+
+set Dimension = 2
+set CFL number                             = 1.0
+set End time                               = 0
+set Start time                             = 0
+set Adiabatic surface temperature          = 0
+set Surface pressure                       = 0
+set Use years in output instead of seconds = false  # default: true
+set Nonlinear solver scheme                = IMPES
+
+
+subsection Boundary temperature model
+  set Model name = box
+  subsection Box
+    set Left temperature = 0
+  end
+end
+
+
+ subsection Initial conditions
+   set Model name = function
+   subsection Function
+    set Function expression = 0
+   end
+ end
+
+
+subsection Gravity model
+  set Model name = vertical
+  subsection Vertical
+    set Magnitude = 0
+  end
+end
+
+
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X extent = 2
+    set Y extent = 1
+  end
+end
+
+
+subsection Initial conditions
+  set Model name = function
+end
+
+
+subsection Material model
+  set Model name = simple
+
+  subsection Simple model
+    set Reference density             = 1
+    set Reference specific heat       = 1
+    set Reference temperature         = 1
+    set Thermal conductivity          = 0
+    set Thermal expansion coefficient = 0
+    set Viscosity                     = 1
+  end
+end
+
+
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 4
+end
+
+
+subsection Model settings
+  set Fixed temperature boundary indicators   = 0, 1
+  # Prescribe a horizontal traction on the vertical boundaries
+  set Prescribed traction boundary indicators = 0 x: function, 1 x: function
+  set Tangential velocity boundary indicators = 
+  set Zero velocity boundary indicators       = 2, 3
+  # Prescribe a zero vertical velocity component on the vertical boundaries
+  set Prescribed velocity boundary indicators = 0 y: function, 1 y: function
+end
+
+
+subsection Boundary traction model
+  subsection Function
+    set Variable names = x,z
+    # select pressure=2 at the left, -2 at the right. this 
+    # corresponds to the following formula for pressure*normal_vector:
```

Comment:
Because the normal vector of the left boundary points in the opposite direction than at the right? If so, please add